commands.json

```diff
@@ -131,6 +131,9 @@
                         "token": "V",
                         "optional": true
                     }
+                ],
+                "command_flags": [
+                    "readonly"
```

Comment:
We need to validate that RedisInsight can handle this

Reply:
Hey @ViktarStarastsenka, can you confirm that this is supported by RedisInsight?

Reply:
it is not supported by now
these command flags will be ignored for now, but I'll raise a ticket to handle them

Reply:
please postpone merging this PR - we'll test it in RedisInsight first

Reply:
tested, you can safely merge the PR